helpers/mime.js

```diff
@@ -2,28 +2,54 @@
 
 const mime = require('mime-types');
 
-const FORCE_EXTENSION_MAPPING = {
-    yidf: 'txt',
-    state: 'txt',
-    diff: 'txt',
-    xml: 'txt' // FIXME: Chrome is not displaying xml files
-};
+const KNOWN_FILE_NAMES_IN_TEXT_FORMAT = ['dockerfile', 'makefile'];
+const KNOWN_FILE_EXTS_IN_TEXT_FORMAT = [
+    'js',
+    'c',
+    'cpp',
+    'cs',
+    'dtd',
+    'h',
+    'm',
+    'java',
```

Comment:
mime returns `'text/x-java-source'` for java, how does the iframe handle this ?

Reply:
do we need this custom mapping ?

Reply:
`iframe` is handled in `displayableMime` (text/html)

Reply:
pushing this to config might be a good idea, considering if we want to add to the list we won't need code changes.